imports.go

```diff
@@ -140,18 +149,20 @@ func (importer *FileImporter) Import(dir, importedPath string) (*ImportedData, e
 	}
 
 	if !found {
-		return nil, fmt.Errorf("Couldn't open import %#v: No match locally or in the Jsonnet library paths.", importedPath)
+		return nil, fmt.Errorf("couldn't open import %#v: no match locally or in the Jsonnet library paths", importedPath)
```

Comment:
I wasn't sure this was a good idea. I was worried I might be breaking parity with C++ Jsonnet. The other issue that many other error messages that don't appear as `fmt.Errorf` parameters weren't picked up.

Ditto below.

Reply:
FYI we don't have error message parity, no need to worry about it. Error messages are not part of the formal spec and may depend on implementation details. So we are not constrained here.

Reply:
For now it is ok, there is no explicit convention yet. Once we decide on some convention, we'll need to document it and keep it consistent everywhere.